ludwig/utils/data_utils.py

```diff
@@ -281,6 +282,22 @@ def load_from_file(file_name, field=None, dtype=int):
     return array
 
 
+def get_filepath_in_diff_format(file_path, desired_format):
+    """
+    Return a file path for a file with same name but different format.
+    a.csv, json -> a.json
+    a.csv, hdf5 -> a.hdf5
+    :param file_path: original file path
+    :param desired_format: desired file format
+    :return: file path with same name but different format
+    """
+    if '.' in desired_format:
+        # Handle the case if the user calls with '.hdf5' instead of 'hdf5'
+        desired_format = desired_format.replace('.', '').strip()
+
+    return os.path.splitext(file_path)[0] + '.' + desired_format
```

Comment:
maybe make it more robust using regex - in case filenames will have additional `.`

Reply:
Never mind, misread.